utils/s3/client.cc

```diff
@@ -286,9 +288,36 @@ client::group_client& client::find_or_create_client() {
 }
 
 future<> client::make_request(http::request req, http::experimental::client::reply_handler handle, std::optional<http::reply::status_type> expected, seastar::abort_source* as) {
-    co_await authorize(req);
     auto& gc = find_or_create_client();
-    co_await gc.retryable_client.make_request(std::move(req), std::move(handle), expected, as);
+    bool renew_creds = false;
+    std::exception_ptr ex;
+    while (true) {
```

Comment:
not sure we need any max retries here, theoretically, it should renew the creds only once and keep going, or terminate the loop by rethrowing any other exception